### PR TITLE
Xrddev 269

### PR DESCRIPTION
**Pull request by Riippi**
Frontend prototype is replaced with new Vue project created with Vue CLI
- All the important pieces from prototype are copied into the new project
- Clients are fetched into the datatable on the second tab
- Datatable in frontpage is using mock data for now

### CODE REVIEW

File: src/proxy-ui-api/frontend/src/components/ClientsDataTable.vue [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+              outline
+              round
+              color="primary"
+              class="text-capitalize table-button"
+              @click="addSubsystem(props.item)"
+            >Add Subsystem</v-btn>
+          </td>
+        </tr>
+      </template>
+      <!--
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/src/components/ClientsDataTable.vue]: some commented out parts, are these meant to be kept or should they be removed?

File: src/proxy-ui-api/frontend/src/store/modules/auth.ts [hunk 2]
```diff
@@ -13,7 +13,15 @@ export const state: AuthState = {
 
 export const getters: GetterTree<AuthState, RootState> = {
   isAuthenticated(state) {
-    return state.authenticated;
+
+    // Check auths state from the cookie
+    // TODO: maybe use localstore instead?
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/src/store/modules/auth.ts]: what jira issue will solve this TODO?

File: src/proxy-ui-api/frontend/src/store/modules/data.ts [hunk 3]
```diff
@@ -2,20 +2,89 @@ import axios from 'axios';
...
+  clients: ClientsArray;
   loading: boolean;
 }
 
 export const state: DataState = {
   cities: [],
+  clients: [
+    {
+      id: 'Member:Dev:Org:111',
+      name: 'Turku',
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/src/store/modules/data.ts]: what is the plan for these, what jira issue will remove this development-code (if any)?

File: src/proxy-ui-api/frontend/src/store/modules/auth.ts (hunk 2 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/store/modules/auth.ts]: This should have been removed. I will clean it.

File: src/proxy-ui-api/frontend/src/store/modules/data.ts (hunk 3 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/store/modules/data.ts]: This mock data is here for filling the client datatable with something.
Will be removed with this:
https://jira.niis.org/browse/XRDDEV-306

File: src/proxy-ui-api/frontend/src/components/ClientsDataTable.vue (hunk 1 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/ClientsDataTable.vue]: I'll remove that

**Pull request merged**